packager/packager.h

```diff
@@ -128,6 +128,11 @@ struct StreamDescriptor {
   std::vector<std::string> dash_accessiblities;
   /// Optional for DASH output. It defines Role elements of the stream.
   std::vector<std::string> dash_roles;
+
+  /// Optional for DASH output. Set to true if only DASH output is needed.
+  bool dash_only = false;
+  /// Optional for HLS output. Set to true if only HLS output is needed.
+  bool hls_only = false;
 };
 
```

Comment:
Can you add a test in packager_test.py?

One option is to modify it from https://github.com/google/shaka-packager/blob/master/packager/app/test/packager_test.py#L652 and use dash_only for audio and use hls_only for video. After writing the test, you can run the below command to generate the golden files for you.

```shell
$ ninja -C out/Debug
$ out/Debug/packager_test.py --test_update_golden_files=1
```

Reply:
Will add the test. Thanks for the commands.